python/apps/taiga/src/taiga/users/api/__init__.py

```diff
@@ -178,6 +183,37 @@ async def update_my_user(request: Request, form: UpdateUserValidator) -> User:
     )
 
 
+#####################################################################
+# delete user
+#####################################################################
+
+
+# TODO
+
+
+#####################################################################
+# delete info user
+#####################################################################
+
+
+@routes.my.get(
+    "/my/user/delete-info",
+    name="my.user.delete-info",
+    summary="Get user delete info",
+    response_model=UserDeleteInfoSerializer,
```

Comment:
This won't work when we get the dependencies updates. Change it better to `response_model=None` and we get avoid a future failure.

Reply:
I think this change should be in the other PR not here.